libdnf/hy-iutil.c

```diff
@@ -531,6 +531,60 @@ what_downgrades(Pool *pool, Id pkg)
     return l;
 }
 
+char *
+pool_solvable_epoch_optional_2str(Pool *pool, Solvable *s, gboolean with_epoch)
+{
+    const char *e;
+    const char *name = pool_id2str(pool, s->name);
+    const char *evr = pool_id2str(pool, s->evr);
+    const char *arch = pool_id2str(pool, s->arch);
+    gboolean present_epoch = FALSE;
+
+    for (e = evr + 1; *e != '-' && *e != '\0'; ++e) {
+        if (*e == ':') {
+            present_epoch = TRUE;
+            break;
+        }
+    }
+    char *output_string;
+    int name_length, evr_length, arch_length;
+    int extra_epoch_length = 0;
+    name_length = strlen(name);
```

Comment:
You can use `int name_length = strlen(name);` and remove line 550. But this is only cosmetic change.